mgr/dashboard: enable different URL for users of browser to Grafana (#37937)
ceph/ceph: doc/cephadm/monitoring.rst

     ceph orch apply grafana 1

> Cephadm handles the Prometheus, Grafana, and Alertmanager configurations

Reviewer: Hmm, how about this?
```suggestion
Cephadm takes care of the configuration of Prometheus, Grafana, and Alertmanager
```

automatically.

> However, there is one exception to this rule. In a somewhat difficult setup, the

Reviewer: ```suggestion
However, there is one exception to this rule. In some setups, the
```

> user's browser might not be able to access the URL to Grafana which is

Reviewer 2: This is somewhat difficult to parse in English. This would be simpler:

`s/the URL to Grafana which is configured/the Grafana URL configured/`

> users browser might not be able to access the URL to Grafana configured in Ceph

Reviewer: ```suggestion
Dashboard user's browser might not be able to access the URL to Grafana configured in Ceph
```

configured in Ceph Dashboard. The cluster being in a different DNS zone than the

> accessing user might be such a scenario.

Reviewer 2: This sentence is awkward, too. How about:

```
One such scenario is when the cluster and the accessing user are each in a different DNS zone.
```

For this case, there is an extra configuration option for Ceph Dashboard, which

> can be used to configure which URL for accessing Grafana by the users browser

Reviewer: ```suggestion
can be used to configure which URL for accessing Grafana by the user´s browser
```
